docs/installation.md

```diff
@@ -218,19 +218,234 @@ On Linux, as an optional post-installation task, you may wish to setup the bot t
+
+general structure of installation command is:
+
+```bash
+conda create -n [name of the environment] [python version] [packages]
+```
+
+so it can be
+
+```bash
+conda create -n freqtrade-conda python=3.8 pandas numpy ta-lib git wheel virtualenv
+```
+
+or if you expect, to use later jupiter for [data-analysis](data-analysis.md), use
+
+```bash
+conda create -n freqtrade-jupyter-conda python=3.8 pandas numpy ta-lib git wheel virtualenv jupyter
+```
+
+the same works for [docker](docker.md), for spyder and other useful programs.
+
+Further read on the topic:
+
+https://towardsdatascience.com/a-guide-to-conda-environments-bc6180fc533?gi=1db972389cd1
+
+https://metager.de/meta/meta.ger3?eingabe=ardsdatascience+guide+to+conda+environment
+
+!!! NOTE:
+It may happen that creating a new Conda environment, populated with selected packages at the moment of creation, takes less time than installing a large, heavy dependent package into the old environment,
+takes less time than installing a large, heavy dependent package into the old environment.
+
+##### Enter/exit freqtrade-conda venv:
+
+to enter choosen conda environment
+
+```bash
+conda activate freqtrade-conda
+
+```
+
+to exit conda environment
+
+```bash
+conda deactivate						
+```
+
+#### 4. Freqtrade Instalation
+
+Within the newly created freqtrade-conda environment, download and install Freqtrade. To process --install command, may take a few minutes
+
+```bash
+# download freqtrade
+git clone https://github.com/freqtrade/freqtrade.git
+
+# enter downloaded directory 'freqtrade'
+cd freqtrade      
+
+# run setup
+./setup.sh --install                                        
+```
+
+Create virtual environment (yes, python3-venv environment, inside conda-environment), where Freqtrade program can run.
+
+Running python3-venv seems to be requirement of the Freqtrade program itself.
+
+```bash
+# create venv environment, inside hidden directory /freqtrede/.env
+python3 -m venv ./.env/
+
+# run the python3-venv environment
+source ./.env/bin/activate
+
+# install last required package            
+pip install -e .						
+```
+
+##### pip install within conda, a reminder:
+
+The documentation of conda says that pip should NOT be used within conda, because internal problems can occur.
+However, they are rare
+
+That is, why the conda-forge channel is preferred: (and if no library is available in conda, you have to use pip)
+
+* more libraries are available (less need for pip)
+* conda-forge works better with pip
+* the libraries are newer
+
+
+#### 5. You are ready
+
+You are ready to run, create the user directory and configuration file, run the program dry to verify that everything is working, and run a backtest to double check.
+
+```bash
+freqtrade create-userdir --userdir user_data
+freqtrade new-config --config config.json
+freqtrade download-data -t 5m 15m 1h --days 100
+freqtrade trade --strategy SampleStrategy  
+freqtrade backtesting -s SampleStrategy
+```
```

Comment:
Personally, i'd only keep the first 2 commands here.

The rest needs more consideration / explanation - it may seem to work like this from a technical point of view, but considering new users (the main audience of a install documentation) this is highly misleading.

consider users which will throw money at this without having developed their strategy yet - and sampleStrategy is not really made to trade right away ...

Reply:
OK